README.md

```diff
@@ -6,8 +6,18 @@ libquicr
 An API library that implements publish/subscribe protocol [draft-ietf-moq-transport-04](https://datatracker.ietf.org/doc/html/draft-ietf-moq-transport-04).
 The API supports both client and server. Server is intended to be implemented as a relay.
 
+API documentation can be found under https://quicr.github.io/libquicr
+
 ## Build
 
+### Dependencies
+
+#### Linux
+
+```
+sudo apt-get update && sudo apt-get install -y gcc g++ golang pkgconf cmake make 
+```
```

Comment:
Might be good to add the Mac dependencies as well

Reply:
will add that now.